jscomp/syntax/ppx_entry.ml

```diff
@@ -329,13 +329,16 @@ let rec unsafe_mapper : Ast_mapper.mapper =
           end             
         |Pexp_constant (Const_string (s, (Some delim))) 
           ->         
-          if Ext_string.equal delim Literals.unescaped_js_delimiter then 
+          if Ext_string.equal delim Literals.unescaped_js_delimiter then
             let s_len  = String.length s in 
             let buf = Buffer.create (s_len * 2) in 
             Ast_utf8_string.check_and_transform loc buf s 0 s_len ;  
             { e with pexp_desc = Pexp_constant (Const_string (Buffer.contents buf, Some Literals.escaped_j_delimiter))}
-          else if Ext_string.equal delim Literals.unescaped_j_delimiter then 
-            Location.raise_errorf ~loc "{j||j} is reserved for future use" 
+          else if Ext_string.equal delim Literals.unescaped_j_delimiter then
+            let starting_loc = {loc with loc_end = loc.loc_start} in
+            let empty_string_concat_exp = {e with pexp_desc = Pexp_constant (Const_string ("", None)); pexp_loc = starting_loc} in
```

Comment:
Why do we want to get rid of it? I think it makes the implementation much simpler if we keep it.